libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/internal/kapt/KaptGenerateStubsTask.kt

```diff
@@ -69,6 +69,9 @@ open class KaptGenerateStubsTask : KotlinCompile() {
             error("KaptGenerateStubsTask.useModuleDetection setter should not be called!")
         }
 
+    @get:Input
+    val verbose = (project.hasProperty("kapt.verbose") && project.property("kapt.verbose").toString().toBoolean() == true)
```

Comment:
:o: This could be modelled as a `Provider` to account for later changes to that property:

```kotlin
project.providers.gradleProperty("kapt.verbose").map { it.toBoolean() }.orElse(false)
```